style.css

```diff
@@ -1,294 +1,283 @@
-    background-color:#FF102D;
-}
-
-.header-image-box{
-    display:flex;
-    justify-content: center;
-    position: relative;
-    z-index: 0;
-    margin-top: -24rem;
-}
-.header-image{
-    width:80%;
-    max-height: 500px;
-}
-
-.buttonBorder{
-    z-index:1;
-    background-color:white;
-    display:flex;
-    justify-content: center;
-    position: absolute;
-    width: 100px;
-    height: 100px;
-    padding: 15px 5px;
-    padding-top: 40px;
-    border-radius:50%;
-    margin-left:-350px;
-    margin-top:30px;
-  }
-  .button.play{
-    display:flex;
-    justify-content: center;
-    border: solid;
-    box-sizing: border-box;
-    width: 10px;
-    height: 14px;
-    border-width: 10px 0px 10px 14px;
-    border-color: transparent transparent transparent #202020;
-  }
-.section-productInfo{
-    margin-top: 2.5%;
-}
-  .link{
-      font-size:1.5rem;
-  }
-
-  .icon-link:hover{
-      color:#FF102D;
-  }
-  
-  .icon-link{
-      text-decoration:none;
-      color: black;      
-  }
-  i{
-      position: absolute;
-  }
-
-  .linkText{
-    margin-left:40px;
-  }
-  .section-productInfo-button{
-      margin-top: 100px;
-  }
-.productInfo-image{
-    margin-top:15%;
-    width: 100%;
-    height: 65%;
-}
-.productInfo-text2{
-    text-align:center;
-    margin-bottom: 20px;
-}
-
-.section-testimonial{
-    margin-top:7%;
-    background-color:#F7F8F8;
-}
-.section-testimonial-image{
-    margin-left:10px;
-}
-.testimonialRow{
-    max-height:30em;
-}
-.testimonial-image{
-    margin:-10px 0;
-    margin-bottom:-15px;
-    max-width:100%;
-    height:auto;
-}
-
-.testimonial-textBox{
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    margin-left:-10px;
-    margin-right:10px;
-    width:100%;
-    height:auto;
-    padding: 15% 15%;
-    font-size: 1.05rem;
-}
-.section-form{
-    background-color:#B9E6FF;
-    margin-top:1%;
-}
-.formText{
-    padding-right:50px;
-    text-align:right;
-}
-
-.form-entry{
-    display:flex;
-    justify-content:center;
-    margin-left:8px;
-    padding-top: 40px;
-    padding-left: 50px;
-}
-.name{
-    margin-left:10px;
-}
-.name-firstName{
-    margin-bottom: -62px;
-}
-.name-lastName{
-    margin-left: 200px;
-}
-.form-input{
-    background-color:#B9E6FF ;
-    border: 0;
-    border-bottom: 1px solid black;
-    height: 2rem;
-    margin-bottom: 30px;
-}
-.long-form{
-    width:103%;
-}
-
-.submit-button{
-    width:30%;
-    margin-left: 265px;
-}
-
-.footer{
-    margin-top: 20px;
-    margin-bottom: 20px;
-    padding-top: 30px;
-    background-color: #F7F8F8;
-    text-align: center;
-}
-.footer-text{
-    color: #FF102D;
-}
-.footer-content{
-    margin-left: 50px;
-    text-align:left;
-}
-h3{
-    margin-left:37px;
-}
-li{
-    list-style:none;
-}
-.logo-icon{
-    margin-top:10px;
-}
-.twitter-icon{
-    margin:0 25px;
-}
-.linkedin-icon{
-    margin:0 50px;
-}
-.bottom-break{
-    height:5px;
+    margin-left:30px;
+}
+.navBar-button:hover{
```

Comment:
Looks like you've got a mix of camelCase and hyphen-case selectors, generally we'd stick to hyphen-case for HTML and CSS, i.e. `nav-bar-button` or `navbar-button` rather than `navBar-button`